src/main/steps/urls.ts

```diff
@@ -23,6 +23,9 @@ export const CERTIFICATE_IN_ENGLISH: PageLink = '/certificate-in-english';
 export const CERTIFIED_TRANSLATION: PageLink = '/certified-translation';
 export const GET_CERTIFIED_TRANSLATION: PageLink = '/get-certified-translation';
 export const COUNTRY_AND_PLACE: PageLink = '/country-and-place';
+export const LIVING_ENGLAND_WALES_TWELVE_MONTHS: PageLink = '/living-england-wales-twelve-months';
+export const INTERSTITIAL: PageLink = '/interstitial';
```

Comment:
This interstitial page looks like it will have the title "You can use English or Welsh courts to get a divorce" (/jurisdiction/interstitial).

Perhaps the naming of this const should resemble the title, so that we know which interstitial page this is.